cypress/e2e/tests/migration/applicationinventory/assessment/miscellaneous.test.ts

```diff
@@ -87,6 +91,38 @@ describe(["@tier3"], "Tests related to application assessment and review", () =>
         );
         applicationList[0].verifyStatus("review", "Not started");
     });
+    it("assess application and overide assessment for that archetype", function () {
+        // Polarion TC MTA-390
+        const archetypesList = [];
+        const tags = createMultipleTags(2);
+        const archetype1 = new Archetype(
+            data.getRandomWord(8),
+            [tags[0].name],
+            [tags[1].name],
+            null
+        );
+        archetype1.create();
+        cy.wait(2 * SEC);
+        archetypesList.push(archetype1);
+        const appdata = {
+            name: data.getAppName(),
+            description: data.getDescription(),
+            tags: [tags[0].name],
+            comment: data.getDescription(),
+        };
+
+        const application1 = new Application(appdata);
+        applicationList.push(application1);
+        application1.create();
+        cy.wait(2 * SEC);
+        archetype1.perform_assessment("low", stakeholderList);
+        application1.clickAssessButton();
+        application1.validateOverrideAssessmentMessage(archetypesList);
+        click(confirmButton);
+        cy.contains("button", "Take", { timeout: 30 * SEC })
+            .should("have.text", "Take")
```

Comment:
Is this validation really needed?

In line 122 you're looking for a button that has the "Take" text, then if you find it, you assert that it has the "Take" text.

If it doesn't have that text it won't be found in the first place

Reply:
yeah sure i can remove this line

Reply:
done